kbase-extension/static/kbase/js/util/jobLogViewer.js

```diff
@@ -1060,39 +1245,38 @@ define([
             });
             fsm.start();
             fsm.bus.on('on-active', () => {
-                startAutoFetch();
+                startLogAutoFetch();
             });
             fsm.bus.on('exit-active', () => {
-                stopAutoFetch();
-            });
-            fsm.bus.on('on-canceled', () => {
-                requestLatestJobLog();
-                stopJobUpdates();
-            });
-            fsm.bus.on('exit-canceled', () => {
-                //  nothing to do?
-            });
-            fsm.bus.on('on-queued', (message) => {
-                doOnQueued(message);
-            });
-            fsm.bus.on('exit-queued', (message) => {
-                doExitQueued(message);
+                stopLogAutoFetch();
+                stopJobStatusUpdates();
             });
-            fsm.bus.on('on-job-not-found', (message) => {
-                doJobNotFound(message);
+            // fsm.bus.on('on-queued', () => {
+            //     doOnQueued();
+            // });
+            // fsm.bus.on('exit-queued', () => {
+            //     doExitQueued();
+            // });
+            fsm.bus.on('on-job-not-found', () => {
```

Comment:
it isn't clear why some changes are attached to FSM transitions and others go in the job status update parsing case cascade from about line 954. I moved things around a bit so at least I could be guaranteed that functions executed in the order I expected them to.